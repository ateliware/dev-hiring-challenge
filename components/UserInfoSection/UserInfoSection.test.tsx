import { render } from "@testing-library/react";
import { UserInfoSection } from ".";

describe("UserInfoSection", () => {
  it("renderiza a seção de informação do autor", () => {
    const wrapper = render(<UserInfoSection />);
    const textElement = wrapper.getByText("hola@facundoleites.com");
    expect(textElement).toBeInTheDocument();
    expect(textElement).toHaveAttribute(
      "href",
      "mailto:hola@facundoleites.com"
    );
    expect(wrapper.container).toMatchSnapshot();
  });
});
