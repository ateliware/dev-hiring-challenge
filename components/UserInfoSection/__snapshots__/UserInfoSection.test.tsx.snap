// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserInfoSection renderiza a seção de informação do autor 1`] = `
<div>
  <div
    class="main"
  >
    <hr
      class="divider"
    />
    <a
      class="mailToButton"
      href="mailto:hola@facundoleites.com"
    >
      hola@facundoleites.com
    </a>
  </div>
</div>
`;
